features/storage/FEATURE_STORAGE/cfstore/source/cfstore_svm.cpp

```diff
@@ -39,7 +44,7 @@ static ARM_DRIVER_STORAGE *cfstore_svm_storage_drv = &ARM_Driver_Storage_MTD_K64
 #endif /* CFSTORE_CONFIG_BACKEND_FLASH_ENABLED */
 
 /* the storage volume manager instance used to generate virtual mtd descriptors */
-static StorageVolumeManager volumeManager;
+StorageVolumeManager volumeManager;
```

Comment:
why was it static previously? shouldn't there be a getter rather than global scope symbol?